spec.bs

```diff
@@ -2288,6 +2297,29 @@ Standard.
      <var ignore>transport</var> to run these steps:
 </div>
 
+The network revocation mechanism requires the following monkeypatch to the [[Reporting]]
+Standard.
+
+<div algorithm=generate-report-patch>
+  Modify the [=generate and queue a report=] algorithm. Add a new step after step 1 that
+  reads:
+
+  2. If |context| is a {{Document}}, and if the result of running [=determine if a navigable has
+      revoked network for itself=] given |context|'s [=node navigable=] is true, then return.
+</div>
+
+The network revocation mechanism requires the following monkeypatch to the [[Webpackage]]
+Standard.
+
+<div algorithm=sxg-patch>
+  Modify the [monkeypatch](https://wicg.github.io/webpackage/loading.html#mp-http-fetch) to [=HTTP
+  fetch=]. Rewrite step 4 under the **"b2" or "b3"** switch case to read:
+
+  4. If the result of determining whether <var ignore>request</var> [=must be blocked due to a
+      revoked partition nonce=] is false, then [=in parallel=], [=wait and queue a report for=] <var
```

Comment:
Indent is wrong here. specfmt's latest version should work here.

Reply:
Ran specfmt on the file.